packages/tocco-util/src/route/asyncRoute.js

```diff
@@ -3,31 +3,21 @@ import React from 'react'
 export default function asyncRoute(getComponent) {
   return class AsyncComponent extends React.Component {
     static Component = null
-    mounted = false
 
     state = {
       Component: AsyncComponent.Component
     }
 
-    componentWillMount() {
+    componentDidMount() {
+      this.mounted = true
```

Comment:
what's that `mounted` variable still needed for?

Reply:
rebase mistake. It isn't.

Reply:
If the component was already mounted and I change the page back from another page it will wait to call setState until the component is mounted.